benchmark/simple/config-burrow.json

```diff
@@ -0,0 +1,32 @@
+{
+  "blockchain": {
```

Comment:
These sections 'blockchain' and 'command' can be moved into the network configuration files.  Then the rest of it seems to be same with other simple's configuration and maybe the file /benchmark/simple/config.yaml could be reused.